### PR TITLE
add dresden.json

### PR DESCRIPTION
**Pull request by RobTranquillo**

### CODE REVIEW

**Comment by johnjohndoe** [comment]
You are missing the changes in the README.md. Please have a look at the contribution guide.

**Comment by RobTranquillo** [comment]
Uh, Iam sorry...
Updated the readme.

**Comment by dirkschumacher** [comment]
There is one point in Mittweida which seems to be quite far away from Dresden.

File: cities/dresden.json [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+            "geometry": {
+                "coordinates": [
+                  13.7596107,
+                  51.1092368
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Markt in Dresden Hellerau",
+                "opening_hours": "Fr 09-16:00",
```
Comment (johnjohndoe) [inline cities/dresden.json]: The time format is not correct here. Should be `Fr 09:00-16:00`.
Please update the commit using `git commit --fixup=d3ee6b` followed by `git rebase -i --autosquash master`. Then update this branch via `git push --force origin market-data-dresden`.

**Comment by johnjohndoe** [comment]
@RobTranquillo Thanks for the corrections. I would like you to put the three "fixup" commits into the initial commit and then update your branch. This can be done as follows:

``` bash
git checkout market-data-dresden
git rebase -i bebad
```

In the editor replace `pick` with `fixup` for the three fixup commits (bottom is the latest).

> pick d3ee6bb add Dresden json
> fixup 6df9db3 add Dresden to supported cities in readme
> fixup 6ae21b2 fix one location and one opening hour
> fixup 1b518f4 fix of the fix (tab to spaces)

Then you save and leave the editor. Afterwards, **one commit** should remain on the branch which contains all your changes. Now you can overwrite the branch on GitHub:

``` bash
git push --force origin market-data-dresden
```

Finally, it is a good idea to `rebase` the branch onto the **latest commit** on `master` (as described in the [contribution guide](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/CONTRIBUTING.md)). This will ensure, there are **no conflicts** with commits which have been done on `master` since you branched off with `market-data-dresden`. Here is what you can do:

First, update the `master` branch on your machine and on your GitHub clone:

``` bash
git checkout master
git pull wo-ist-markt master
git push origin master
```
Then `rebase`  your branch onto `master` and update it on GitHub once again:

``` bash
git rebase master market-data-dresden
git push --force origin market-data-dresden
```

This will automatically update the pull request since the branch name stayed the same.

---

As always, I will answer questions in case your have any.

**Comment by johnjohndoe** [comment]
Something went wrong. I fixed the branch on my local copy and merged it into `master`. You can throw away your local branch and the corresponding branch on your GitHub clone now.

To sum it up: Welcome on the map :tada:
Please have a look at #38.

**Pull request closed without merging**